rest-api/config/options/all.xml

```diff
@@ -130,6 +130,106 @@
+        </group>
+        <group label="Documents">
+          <match pattern="application/msword"/>
+          <match pattern="application/vnd.wordperfect"/>
+          <match pattern="application/x-wordstar"/>
+          <match pattern="application/pdf"/>
+          <match pattern="application/postscript"/>
+          <match pattern="application/rtf"/>
+          <match pattern="application/x-xywrite"/>
+          <match pattern="application/x-mass11"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.wordprocessingml.document"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.wordprocessingml.template"/>
+          <match pattern="application/vnd.ms-word.document.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-word.template.macroEnabled.12"/>
+        </group>
+        <group label="Spreadsheets">
+          <match pattern="application/vnd.ms-excel"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.spreadsheetml.template"/>
+          <match pattern="application/vnd.ms-excel.sheet.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-excel.template.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-excel.addin.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-excel.sheet.binary.macroEnabled.12"/>
+        </group>
+        <group label="Presentations">
+          <match pattern="application/vnd.ms-powerpoint"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.presentationml.presentation"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.presentationml.template"/>
+          <match pattern="application/vnd.openxmlformats-officedocument.presentationml.slideshow"/>
+          <match pattern="application/vnd.ms-powerpoint.addin.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-powerpoint.presentation.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-powerpoint.template.macroEnabled.12"/>
+          <match pattern="application/vnd.ms-powerpoint.slideshow.macroEnabled.12"/>
+        </group>
+        <group label="Other Documents">
+          <match pattern="application/acad"/>
+          <match pattern="application/ms-project"/>
+          <match pattern="application/ms-works"/>
+          <match pattern="application/vnd.ms-works"/>
+          <match pattern="application/vnd.visio"/>
+          <match pattern="application/x-microsoft-publisher"/>
+          <match pattern="application/x-uniplex"/>
+        </group>
+        <group label="Business Cards">
+          <match pattern="text/x-vcard"/>
+        </group>
+        <group label="Email &amp; Mailboxes">
+          <match pattern="application/EML"/>
+          <match pattern="application/mbox"/>
+        </group>
+        <group label="Others">
+          <match pattern=".+"/>
```

Comment:
show-remaining includes records without attachments/media which doesn't really fit. Not sure on how ml-constraints handles this yet, but the goal is to group the remaining valid and unknown mime-types here.